### PR TITLE
[SAGE-10] spacing token update

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] update spacing tokens in sage
- [x] resolve followup issues: `Card List`, `Panel List`, `Table`, `Panel Figure`, `Stat Box`

#### Cards
|  Before  |  After  |
|--------|--------|
|Screen Shot 2022-01-26 at 1 14 37 PM|Screen Shot 2022-01-26 at 1 13 56 PM|

#### Panels
|  Before  |  After  |
|--------|--------|
|Screen Shot 2022-01-26 at 1 15 35 PM|Screen Shot 2022-01-26 at 1 15 20 PM|

## Testing in `sage-lib`

Visit the following react or rails components:
- Card
- Panel
- Panel figure
- Table
- Stat Box

## Testing in `kajabi-products`

1. (**HIGH**) Update spacing values for all Cards and Panels.
    - [ ] Panel list - Products index
    - [ ] Table - People Index
    - [ ] Card - Newsletters Edit
    - [ ] Panel - Product Index
    - [ ] Stat box - ?

## Related

Closes [SAGE-10](https://kajabi.atlassian.net/browse/SAGE-10)

### CODE REVIEW

**Comment by monicawheeler** [comment]
@QuintonJason - when you have a moment, could you update the branch to merge into to be `SAGE-19_EPIC-foundations` rather than `develop`? Thank you!

**Comment by monicawheeler** [comment]
@QuintonJason - be sure to include some details in the "Testing in kajabi-products" section on how QE can locate areas of the app to test these changes. Thank you!

**Review by monicawheeler — APPROVED** [review]
Looks great! One minor note about QE testing, but not a blocker on merging

File: packages/sage-assets/lib/stylesheets/components/_panel_figure.scss [hunk 1]
```diff
@@ -59,21 +59,21 @@ $-panel-figure-default-background-color: sage-color(grey, 100);
 }
 
 .sage-panel__figure--bleed-bottom {
-  width: calc(100% + #{2 * sage-spacing()});
-  margin: 0 (-1 * sage-spacing()) (-1 * sage-spacing());
+  width: calc(100% + #{2 * sage-spacing(lg)});
```
Comment (monkeypox8) [inline packages/sage-assets/lib/stylesheets/components/_panel_figure.scss:62]: Just a thought; I'm wondering if we'd want to assign the bleed width calculation as a component Sass var. The three variations appear to share the same value.

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM! Tested with the bridge on a few views, looks great!

File: docs/app/views/pages/spacing.html.erb [hunk 2]
```diff
@@ -5,6 +5,13 @@
 )) %>
 <% end %>
 
+<%= md("
+## Coloring Type
+
+While applying colors to type on-the-fly should generally be avoided
```
Comment (philschanely) [inline docs/app/views/pages/spacing.html.erb:11]: Am I missing context or is this new text an oversight?

File: packages/sage-assets/lib/stylesheets/components/_panel_figure.scss (hunk 1 above)
Comment (philschanely) [reply packages/sage-assets/lib/stylesheets/components/_panel_figure.scss:62]: Thats a good idea: it should be tied to whatever the current side padding is...

**Review by philschanely — APPROVED** [review]
Small items but 👍  otherwise!

**Pull request merged**